tests/kafkatest/benchmarks/streams/streams_simple_benchmark_test.py

```diff
@@ -50,27 +54,38 @@ def test_simple_benchmark(self, test, scale):
         self.zk = ZookeeperService(self.test_context, num_nodes=1)
         self.zk.start()
         self.kafka = KafkaService(self.test_context, num_nodes=scale, zk=self.zk, version=DEV_BRANCH, topics={
-            'simpleBenchmarkSourceTopic' : { 'partitions': scale, 'replication-factor': self.replication },
-            'countTopic' : { 'partitions': scale, 'replication-factor': self.replication },
-            'simpleBenchmarkSinkTopic' : { 'partitions': scale, 'replication-factor': self.replication },
-            'joinSourceTopic1KStreamKStream' : { 'partitions': scale, 'replication-factor': self.replication },
-            'joinSourceTopic2KStreamKStream' : { 'partitions': scale, 'replication-factor': self.replication },
-            'joinSourceTopic1KStreamKTable' : { 'partitions': scale, 'replication-factor': self.replication },
-            'joinSourceTopic2KStreamKTable' : { 'partitions': scale, 'replication-factor': self.replication },
-            'joinSourceTopic1KTableKTable' : { 'partitions': scale, 'replication-factor': self.replication },
-            'joinSourceTopic2KTableKTable' : { 'partitions': scale, 'replication-factor': self.replication },
-            'yahooCampaigns' : { 'partitions': 20, 'replication-factor': self.replication },
-            'yahooEvents' : { 'partitions': 20, 'replication-factor': self.replication }
+            'simpleBenchmarkSourceTopic1' : { 'partitions': scale, 'replication-factor': self.replication, "config": "message.timestamp.type=LogAppendTime" },
+            'simpleBenchmarkSourceTopic2' : { 'partitions': scale, 'replication-factor': self.replication, "config": "message.timestamp.type=LogAppendTime" },
+            'simpleBenchmarkSinkTopic' : { 'partitions': scale, 'replication-factor': self.replication, "config": "message.timestamp.type=LogAppendTime" },
+            'yahooCampaigns' : { 'partitions': 20, 'replication-factor': self.replication, "config": "message.timestamp.type=LogAppendTime" },
+            'yahooEvents' : { 'partitions': 20, 'replication-factor': self.replication, "config": "message.timestamp.type=LogAppendTime" }
         })
         self.kafka.log_level = "INFO"
         self.kafka.start()
- 
+
+        self.key_skew = 0
+        self.value_size = 1024
+
+        load_test = ""
+        if test in STREAMS_SIMPLE_TEST:
+            load_test = "load-one"
+        if test in STREAMS_JOIN_TEST:
+            load_test = "load-two"
+        if test in NON_STREAMS_TEST:
+            load_test = "load-one"
```

Comment:
Would it be the acme of foolishness to suggest that we just always "load-two"?

Then, we wouldn't have to maintain the extra lists of tests at the top of the file, and we wouldn't need this condition here.
And also, we'd eliminate a difference among the benchmarks, since the brokers would be loaded with the same dataset in all cases (even if one of them is unused).

Reply:
Alternatively, we can get rid of those lists by just matching on whether the test name ends with "join" (we match on table name elsewhere in this PR)

Reply:
Yes I'm thinking about the same thing, since the current test spend up to half of the time loading the data, which could be costly to run in aws. Will update the JIRA accordingly

Reply:
Ah, I didn't realize load takes that long, but in retrospect, it should have been obvious. Then, of course we shouldn't always load both, and my suggestion is just to match on the test name.